docs/sphinx/using/simulators.rst

```diff
@@ -115,3 +113,26 @@ options to :code:`nvq++`
 .. code:: bash 
 
     nvq++ --target tensornet src.cpp ...
+
+
+Default Simulator
+==================================
+If no explicit target is set, i.e. if the code is compiled without any :code:`--target` flags, then CUDA Quantum makes a default choice for the simulator.
+
+If an NVIDIA GPU and CUDA runtime libraries are available, the default target is set to `nvidia`. This will utilize the cuQuantum single-GPU simulator. :ref:`cuQuantum single-GPU` 
```

Comment:
```suggestion
If an NVIDIA GPU and CUDA runtime libraries are available, the default target is set to `nvidia`. This will utilize the :ref:`cuQuantum single-GPU state vector simulator <cuQuantum single-GPU>`.  
```